docs/production/authentication-methods.md

```diff
@@ -145,6 +145,9 @@ All of these data synchronization options have the same model:
   name/avatar/etc. from LDAP (as configured) on account creation.
 * The `manage.py sync_ldap_user_data` cron job will automatically
   update existing users with any changes that were made in LDAP.
+* On every login, existing users are updated with any changes that were made in
+  LDAP, even if a syncing cron job has not been setup. These updates can be
+  turned off by setting `AUTH_LDAP_ALWAYS_UPDATE_USER = False`.
```

Comment:
So one thing that I'm not sure about: I **think** that this only applies to the `full_name` field, and not anything else, since it's our code that does the syncing for everything else.  

Given that, it might be cleaner to just turn off that upstream setting?  I think that would have everything be consistent with our documentation without having to do anything crazy...

Reply:
Oh, good catch, let me take a look!

Reply:
It [looks at the USER_ATTR_MAP and tries to map to the corresponding fields](https://github.com/django-auth-ldap/django-auth-ldap/blob/master/django_auth_ldap/backend.py#L619), which only works for the `full_name` and `shortname` fields, I guess. `avatar` and custom fields don't get synced. 

I agree -  turning off the setting, with a note in the update notes should be good enough.